### PR TITLE
Update License-Management.md

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by psmulovics** [comment]
"10:30:55 AM: TypeError: Cannot read properties of undefined (reading 'sort')"

File: docs/bok/Activities/Level-2/License-Management.md [hunk 1]
```diff
@@ -131,6 +131,11 @@ This includes [CopyLeft](../../Artifacts/Licenses#3-copyleft-licenses) licenses
 
 Using the correct licenses will form part of License Policy and it will be necessary to ensure developers in your organisation are aware of the limitations around the use of open source.  Although these limitations will vary from firm-to-firm, the Linux Foundation provides several training courses suitable for orienting developers on their responsibilities relating to open source licensing:
 
+**See:**
+
+<BokTagList tag="License Compliance Management (Role)" filter="Training" />
```
Comment (psmulovics) [inline docs/bok/Activities/Level-2/License-Management.md:136]: supposed to be 
License Compliance Management (Activity)

**Review by psmulovics — CHANGES_REQUESTED** [review]
You have to fix that line

**Comment by psmulovics** [comment]
@robmoffat

**Comment by robmoffat** [comment]
done

**Review by psmulovics — APPROVED** [review]

**Review by GITHUB01reo — COMMENTED** [review]
Hadir

**Review by GITHUB01reo — COMMENTED** [review]
Approved

**Pull request merged**